extended_euclidean_gcd/Python/extended_euclidean_gcd.py

```diff
@@ -0,0 +1,29 @@
+# Element style: gcd(a,b)-> a*x + b*y = r = gcd(a,b)
```

Comment:
@thekaleidoscope, this needs to done in euclidean_gcd/Python/euclidean_gcd.py only. Not in the different file.